initfiles/sbin/hpcc-run.sh.in

```diff
@@ -52,134 +52,143 @@ getIPS(){
 }
 
 getDali(){
-        DIP=`${INSTALL_DIR}/sbin/configgen -env ${envfile} -listall | grep Dali | awk -F, '{print \$3}'  | sort | uniq`
+    DIP=`${INSTALL_DIR}/sbin/configgen -env ${envfile} -listall | grep Dali | awk -F, '{print \$3}'  | sort | uniq`
 }
 
 createIPListFile(){
-    _file=$1
-    echo "$IPS" > $_file
+    local _output=$1
+    echo "$IPS" > $_output
 }
 
 createIPListFileExcludeDIP(){
-  _file=$1
-  echo "$IPS" | grep -v $DIP  > $_file
+    local _input=$1
+    local _output=$2
```

Comment:
Yeah that's one of the reasons why I originally went with matching against string literals instead of using regular expressions.  If I have a DIP of `10.20.8.17` it will match `10A20.8.17` where as if we just match against literals, we don't have the issue.  Cons:  it isn't as pretty.  Pros:  it just works.